tests/h/presenters/annotation_json_test.py

```diff
@@ -1,208 +1,119 @@
 import datetime
-from unittest import mock
 from unittest.mock import create_autospec
 
 import pytest
+from h_matchers import Any
 from pyramid import security
-from pyramid.authorization import ACLAuthorizationPolicy
 
 from h.formatters import AnnotationFlagFormatter
 from h.presenters.annotation_json import AnnotationJSONPresenter
-from h.security.permissions import Permission
 from h.traversal import AnnotationContext
 
 
 class TestAnnotationJSONPresenter:
-    def test_asdict(self, document_asdict, groupfinder_service, links_service):
-        ann = mock.Mock(
-            id="the-id",
-            created=datetime.datetime(2016, 2, 24, 18, 3, 25, 768),
-            updated=datetime.datetime(2016, 2, 29, 10, 24, 5, 564),
-            userid="acct:luke",
-            target_uri="http://example.com",
-            text="It is magical!",
-            tags=["magic"],
-            groupid="__world__",
-            shared=True,
-            target_selectors=[{"TestSelector": "foobar"}],
-            references=["referenced-id-1", "referenced-id-2"],
-            extra={"extra-1": "foo", "extra-2": "bar"},
+    def test_asdict(self, annotation, context, DocumentJSONPresenter):
+        annotation.created = datetime.datetime(2016, 2, 24, 18, 3, 25, 768)
+        annotation.updated = datetime.datetime(2016, 2, 29, 10, 24, 5, 564)
+        annotation.references = ["referenced-id-1", "referenced-id-2"]
+        annotation.extra = {"extra-1": "foo", "extra-2": "bar"}
```

Comment:
Set the minimum we have to locally, and then do everything else by reference.

Reply:
Certainly good to see `mock.Mock()` gone